src/Init/Data/BitVec/Lemmas.lean

```diff
@@ -2351,6 +2354,20 @@ theorem toNat_shiftConcat_lt_of_lt {x : BitVec w} {b : Bool} {k : Nat}
   have := Bool.toNat_lt b
   omega
 
+theorem getElem_shiftConcat (x : BitVec w) (b : Bool) (h : i < w) :
```

Comment:
```suggestion
theorem getElem_shiftConcat {x : BitVec w} {b : Bool} (h : i < w) :
```

We're trying to move to mathlib-style implicits, where arguments that can be inferred from the rewrite are implicit.